testdata/goldens/go/index.html

```diff
@@ -8,7 +8,9 @@
     {% verbatim %}
     <div>
       <article data-uid="cloud.google.com/go/storage">
-<h1 class="page-title">Package cloud.google.com/go/storage
+<h1 class="page-title">

```

Comment:
Any way we can avoid these whitespace changes? This would modify every file. Would be nice not to. Fine if not.

Reply:
Yes, if we keep it all in one line in the template, but it's slightly less readable:

```<h1 class="page-title">{{#friendlyApiName}}{{friendlyApiName}} - {{/friendlyApiName}}{{#title}}{{title}}{{/title}}{{^title}}{{>partials/title}}{{/title}}</h1>```

Wdyt?

Reply:
That is pretty intense to read. So, let's leave it as-is.